jdi-light-angular/src/main/java/com/epam/jdi/angular/asserts/AutoCompleteAssert.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.jdi.angular.asserts;
+
+import com.epam.jdi.angular.elements.complex.*;
+import com.epam.jdi.light.asserts.generic.*;
+import com.epam.jdi.light.common.*;
+import org.hamcrest.*;
+import org.hamcrest.Matcher;
+
+import java.util.*;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.*;
+
+public class AutoCompleteAssert extends UIAssert<AutoCompleteAssert, AutoComplete> implements ITextAssert<AutoCompleteAssert> {
+
+    @JDIAction("Assert that '{name}' is expanded")
+    public AutoCompleteAssert expanded() {
+        jdiAssert(element.expanded().toString(), Matchers.is("true"));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' is collapsed")
+    public AutoCompleteAssert collapsed() {
+        jdiAssert(element.collapsed().toString(), Matchers.is("true"));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' text {0}")
+    @Override
+    public AutoCompleteAssert text(String expected) {
+        jdiAssert(element().getValue(), Matchers.is(expected));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' text {0}")
+    @Override
+    public AutoCompleteAssert text(Matcher<String> condition) {
+        jdiAssert(element().getValue(), condition);
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has placeholder {0}")
+    public AutoCompleteAssert placeholder(String expected) {
+        jdiAssert(element.placeholder(), Matchers.is(expected));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name} is mandatory")
+    public AutoCompleteAssert mandatory() {
+        jdiAssert(element.isMandatory().toString(), Matchers.is("true"));
```

Comment:
Is toString() necessary here? Can we compare Boolean values? (And in other similar places)

Reply:
I tried to compare boolean values, however matcher requires object, not a primitive, hence conversion to string.

Reply:
Found a way to use booleans.